Jenkinsfile_updatecli

```diff
@@ -21,13 +21,27 @@ pipeline {
     AWS_ACCESS_KEY_ID     = credentials('packer-aws-access-key-id')
     AWS_SECRET_ACCESS_KEY = credentials('packer-aws-secret-access-key')
     AWS_DEFAULT_REGION    = 'us-east-2'
+    UPDATECLI_AZURE = credentials('updatecli-azure-serviceprincipal')
   }
   stages {
     stage('Check Configuration Update') {
       // Run updatecli's diff on both push and pull requests (in case a configuration change breaks updatecli)
       steps {
         catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-          updatecli(action: 'diff', cronTriggerExpression: cronExpr)
+          script {
+            withCredentials([
+              usernamePassword(
+                credentialsId: 'github-app-updatecli-on-jenkins-infra',
+                usernameVariable: 'USERNAME_VALUE', // Setting this variable is mandatory, even if of not used when the credentials is a githubApp one
+                passwordVariable: 'UPDATECLI_GITHUB_TOKEN'
+              )
+            ]) {
```

Comment:
as discussed together, these instructions duplicates the 
```
UPDATECLI_AZURE = credentials('updatecli-azure-serviceprincipal')
```

instruction

Reply:
as seen together, not as simple, we still need both for now.